src/cppwinrt.build.pre.props

```diff
@@ -31,13 +31,17 @@
   <!-- This is magic that tells msbuild to link against the Desktop platform
        instead of the App platform. This you definitely want, because we're not
        building a true universal "app", we're building a desktop application
-       with xaml. Without this, we'll link against the app CRT, and the
-       unpackaged story _will not_ work. Furthermore, APIs like
-       CreatePseudoConsole won't be linkable without this.
+       with xaml. Furthermore, APIs like CreatePseudoConsole won't be linkable without this.

```

Comment:
> Furthermore, APIs like CreatePseudoConsole won't be linkable without this.

Probably should yank this text, since we won't be able to link it anymore (but we don't _need_ to anymore)

Reply:
Ah yes. I probably will just change it to say "APIs like CreatePsuedoConsole aren't necessarily linkable without the correct partition, but we have our own copy in winconpty."

Just in case we have to come back past this part. Editing linking and MSBuild is SO PAINFUL that I want to leave ample records.